src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiFrameUI.java

```diff
@@ -47,7 +47,7 @@ public void renderComponent(Frame component, FontRenderer fontRenderer) {
 
         glColor4f(.17f,.17f,.18f,.9f);
         RenderHelper.drawFilledRectangle(0,0,component.getWidth(),component.getHeight());
-        glColor3f(.59f,.05f,.11f);
+        glColor3f(.60f,.56f,1.00f);
```

Comment:
Found by using RGB percentages values.
(RGB Singular Code / 256) * 100 = x
x MOD 1 = .xf

Reply:
Yes, this is always the case, OpenGL expects values from 0-1, as explained here: https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glColor.xml

> Neither floating-point nor signed integer values are clamped to the range 0 1 before the current color is updated. However, color components are clamped to this range before they are interpolated or written into a color buffer.